packages/npm-dependencies-validation/test/utils/packageJsonUtil/packageJsonUtil.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from "chai";
+import { VscodeFsUri, VscodeWsFolder } from "../../../src/types";
+import { isManagedByNpm6 } from "../../../src/utils/packageJsonUtil";
+
+describe("packageJsonUtil unit test", () => {
+  it("not manged by npm, has pnpm-workspace.yaml", async () => {
+    const uri: VscodeFsUri = {
+      fsPath:
+        "test/utils/packageJsonUtil/projects/pnpm-workspace/wsFolder1/child/package.json",
```

Comment:
consider placing the "samples" closer to the tests (e.g in sibling folder)
instead of one single "samples" (projects) folder.
it makes it harder to understand which samples are redundant.

Reply:
the samples are under specific test folder: test/utils/packageJsonUtil/projects

Reply:
I meant a test sub-folder per test
so if we delete a test in the future we will also know which sub-folder to delete.